client/src/crypto/mod.rs

```diff
@@ -2,6 +2,9 @@
 mod native;
 #[cfg(feature = "native_crypto")]
 pub(crate) use native::*;
+#[cfg(feature = "unstable")]
+#[cfg_attr(docsrs, doc(cfg(feature = "unstable")))]
+pub use native::*;
```

Comment:
No, duplicate what you did for openssl.

Reply:
Got it, thanks!

Reply:
Sorry. This could be trivial. But, I don't get it :)

I tried the following and the compiler is not happy,
```
+#[cfg(feature = "unstable")]
+#[cfg_attr(docsrs, doc(cfg(feature = "unstable")))]
+pub use self::openssl::*;
```

```
error[E0432]: unresolved import `self::openssl`
  --> client/src/crypto/mod.rs:15:15
   |
15 | pub use self::openssl::*;
   |               ^^^^^^^ could not find `openssl` in `self`

```

Reply:
Handled that for you in https://github.com/bilelmoussaoui/oo7/pull/92

Reply:
Thank you :)